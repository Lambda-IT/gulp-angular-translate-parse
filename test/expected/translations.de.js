var Translations; (function (Translations) {var de = {"ns":{"test_key":"ns.test_key"},"ns1":{"ns2":{"test_key":"test text"},"ns3":{"test_key":"Here there is already text inside"}},"template":{"ns2":{"test_key":"test text"},"ns3":{"test_key":"Here there is already text inside"},"test_key":"template.test_key"},"test_key":"test_key"};Translations.de = de; })(Translations || (Translations = {}));