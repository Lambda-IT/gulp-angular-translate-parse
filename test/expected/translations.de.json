{"test_key":"test_key","ns":{"test_key":"ns.test_key"},"ns1":{"ns2":{"test_key":"ns1.ns2.test_key"},"ns3":{"test_key":"ns1.ns3.test_key"}}}